package.json

```diff
@@ -8,9 +8,12 @@
   "scripts": {
     "bump-gradle-version": "node ./scripts/gradle-version-update.js --package-version=${npm_package_version} && git add app/build.gradle",
     "build": "./gradlew clean assembleDebug && npm run move-apk",
+	"build:win": "gradlew clean assembleDebug && npm run create-folder && npm run copy-apk",
```

Comment:
Please keep the same indentation

Reply:
I don't know why, but on my VS and NotePad++ it shows Fine, only on gitHub the indentation is wrong, so I fixed directly on Github